app/src/main/java/org/stepik/android/view/step/mapper/NavigationActionMapper.kt

```diff
@@ -22,4 +27,54 @@ constructor() {
         isAutoplayEnabled: Boolean = false
     ): StepNavigationAction =
         StepNavigationAction.ShowLesson(direction, lessonData, isAutoplayEnabled)
+
+    fun mapToRequiredSectionAction(
```

Comment:
`NavigationActionMapper` should be in `presentation` layer as it was used from presenter